package.json

```diff
@@ -28,6 +28,7 @@
     "@aws-sdk/client-dynamodb": "^3.44.0",
     "@aws-sdk/client-lambda": "^3.27.0",
     "@aws-sdk/client-s3": "^3.27.0",
+    "@aws-sdk/client-sqs": "^3.7.0",
```

Comment:
Is there any need to keep our `@aws-sdk` packages on the same version? We now have `3.44.0`, `3.27.0`, and `3.7.0`. Not sure if this could be a problem somewhere along the line.

Reply:
Each client is separate. we should probably be looking at upgrading the clients on the NextJS side